src/nvim/terminal.c

```diff
@@ -233,6 +233,8 @@ Terminal *terminal_open(buf_T *buf, TerminalOptions opts)
   RESET_BINDING(curwin);
   // Reset cursor in current window.
   curwin->w_cursor = (pos_T){ .lnum = 1, .col = 0, .coladd = 0 };
+  // Intitialzie to check if the scrollback buffer has been allocated inside a TermOpen autocmd
```

Comment:
- spellcheck
- use punctuation in comments, it avoids ambiguity.

```suggestion
  // Initialize in case the scrollback buffer has been allocated inside a TermOpen autocmd.
```